simpleclient_servlet/src/main/java/io/prometheus/client/filter/MetricsFilter.java

```diff
@@ -0,0 +1,128 @@
+package io.prometheus.client.filter;
+
+import io.prometheus.client.Histogram;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
simpleclient_servlet is for exposing metrics to Prometheus, it shouldn't be pulling in something like apache commons.

This probably belongs in a separate module.